src/codelabs/dart-cheatsheet.md

```diff
@@ -1,18 +1,20 @@
 ---
 title: Dart cheatsheet codelab
 description: Interactively learn (or relearn) some of Dart's unique features.
+js: [{url: 'https://dartpad.dev/inject_embed.dart.js', defer: true}]
 ---
-
+<?code-excerpt replace="/ *\/\/\s+ignore_for_file:[^\n]+\n//g; /(^|\n) *\/\/\s+ignore:[^\n]+\n/$1/g; /(\n[^\n]+) *\/\/\s+ignore:[^\n]+\n/$1\n/g"?>
```

Comment:
I have the same question asked in [this comment](https://github.com/dart-lang/site-www/pull/2492#discussion_r445183909) - what's this doing?